### PR TITLE
[FIX] website, website_sale: uniformize country_infos

### PR DESCRIPTION
**Pull request by JKE-be**
In commit 79807d6, we had the notion of zip_required/state_required.
Add it to website controller too.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by JKE-be** [comment]
@robodoo delegate=rdeodoo

**Comment by JKE-be** [comment]
Friendly Reminder @rdeodoo

**Comment by rdeodoo** [comment]
> Friendly Reminder @rdeodoo

I sent you the review on discord the other day :)

File: addons/website/controllers/main.py [hunk 1]
```diff
@@ -159,9 +159,14 @@ def change_lang(self, lang, r='/', **kwargs):
         return redirect
 
     @http.route(['/website/country_infos/<model("res.country"):country>'], type='json', auth="public", methods=['POST'], website=True)
-    def country_infos(self, country, **kw):
-        fields = country.get_address_fields()
-        return dict(fields=fields, states=[(st.id, st.name, st.code) for st in country.state_ids], phone_code=country.phone_code)
+    def country_infos(self, country, mode, **kw):
```
Comment (JKE-be) [inline addons/website/controllers/main.py:162]: mode is wrong here

**Comment by JKE-be** [comment]
14.4 is deprecated

**Pull request closed without merging**